crates/api/src/peer_meta_store.rs

```diff
@@ -1,34 +1,34 @@
-use crate::{builder, config, AgentId, BoxFut, K2Result, SpaceId, Timestamp};
+use crate::{builder, config, BoxFut, K2Result, SpaceId, Timestamp, Url};
 use futures::future::BoxFuture;
 use std::sync::Arc;
 
 /// A store for peer metadata.
 ///
-/// This is expected to be backed by a key-value store that keys by space, agent_id and key.
+/// This is expected to be backed by a key-value store that keys by space, peer URL and key.
```

Comment:
This was my mistake implementing this on the first pass. We want to track interactions with a peer/node rather than each agent on that node. Here, it's not very interesting what the set of agents on that machine are doing, really just their common arc set.

What is a pain is that this is a session value. So every time the peer restarts or reconnects we are starting fresh with information about them... I think that's fine for gossip, and probably for fetch too. I'm not sure whether it's a good enough replacement for the information we were capturing in the peer metrics in K1. I'm thinking we'll know when we need something like that!

Reply:
Indeed, the legacy peer metrics never really worked as intended. We'll need to design that fresh when we get to sharding I think.